### DISCUSSION TITLE
Re-verification student status failure with location error message despite enabling browser's and system's location

### DISCUSSION THREAD

**Discussion by hlleung74** [community/community#54642] [GitHub Education]
### Hi everyone,

I received the following location error message when renewing my student status.

> Your application cannot be reviewed until you fix the following:
> Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.

I have tried every solution I can find online to resolve this, but nothing seems to work.

I tried to submit my applications via my laptop (macOS Monterey) and mobile (iOS). I tried different browsers, including Google Chrome, Firefox, and Microsoft Edge. At the same time, I ensured that I enabled both the browser and system settings for the location.

Any suggestion on how to solve this issue? Thanks!!!

The screenshots below illustrate the location settings of my laptop:

### System's Location Setting
<img width="648" alt="System Location Setting" src="https://user-images.githubusercontent.com/44010460/236640183-6a1d3684-fe1c-478d-9889-4bd938baa706.png">

### Google Chrome's Location Setting
<img width="648" alt="Google Chrome Location Setting" src="https://user-images.githubusercontent.com/44010460/236640192-53969904-bf55-42d8-b9b0-a1193e332f9a.png">

### Firefox's Location Setting
<img width="648" alt="Firefox Location Setting" src="https://user-images.githubusercontent.com/44010460/236640205-c9273fe2-e7a5-4d3d-bb44-f21db0e999ac.png">

### Microsoft Edge's Location Setting
<img width="648" alt="Microsoft Edge Location Setting" src="https://user-images.githubusercontent.com/44010460/236640219-ea59beb3-6244-417c-834d-36054bcf3f42.png">

**Comment by iamreiyn** [comment]
Hello, @hlleung74 

Have you tried to cancel your current application and create a new one after the location permissions have been enabled and checked?
